e2e/token-vault-app/src/main.ts

```diff
@@ -1,40 +1,50 @@
-import './app/app.element';
-import { client } from '@forgerock/token-vault';
 import {
   Config,
   FRUser,
   TokenManager,
   UserManager,
 } from '@forgerock/javascript-sdk';
-// Initialize the token vault client
+import { client } from '@forgerock/token-vault';
+
+/** ****************************************************
+ * START: CONFIGURE THE TOKEN VAULT
+ **************************************************** */
+
+// Initialize the Token Vault Client
 const register = client({
   app: {
-    origin: 'http://localhost:8000',
-    url: 'http://localhost:8000',
+    origin: 'http://localhost:5823',
   },
   interceptor: {
-    file: new URL('../interceptor.ts', import.meta.url).pathname,
+    file: new URL('/interceptor.ts', import.meta.url).pathname,
   },
   proxy: {
-    origin: 'http://localhost:9000',
-    url: 'http://localhost:9000',
+    origin: 'http://localhost:5833',
   },
 });
 
-// Register the interceptor
+// Register the Token Vault Interceptor
 const interceptor = await register.interceptor();
 
-// Register the proxy
+// Register the Token Vault Proxy
 const proxy = await register.proxy(
   document.getElementById('token-vault') as HTMLElement
 );
 
-// Register the token store replacement
-const storeReplacement = register.store();
+// Register the Token Vault Store
+const tokenStore = register.store();
```

Comment:
Token Vault Client initialization and registration of Interceptor (service worker), Proxy (iframe) and Store. The values returned to `interceptor` and `proxy` are optional and used only if the dev will want to "destroy" the iframe or unregister the service worker.